.github/workflows/actionlint.yml

```diff
@@ -0,0 +1,68 @@
+name: actionlint
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+on:
+  pull_request:
+    branches:
```

Comment:
shouldn t we link on dev-aa.bb.x branches as well ? 
same for aa.bb.y as well ?